packages/bpk-component-map/README.md

```diff
@@ -111,6 +119,7 @@ When using `withGoogleMapsScript`, some additional props are available:
 | className        | string                                       | false                    | null                 |
 | large            | bool                                         | false                    | false                |
 | onClick          | func                                         | false                    | null                 |
+| selected         | bool                                         | false                    | false
```

Comment:
Nitpicking: the last `|` is missing :-P

On a personal project I use prettier to format `md` files in a pre-commit hook, maybe you would be interested in adding something like that:
https://github.com/sogame/danger-plugin-toolbox/blob/master/package.json#L35-L38

Reply:
Ooh thanks, good tip!